docs/src/idiomatic/003record-sets.md

```diff
@@ -0,0 +1,119 @@
+# Idiom 2: Replace sets of mixed records with disjoint unions
```

Comment:
Why is it called Idiom 2? We have a list of 14 idioms: https://apalache.informal.systems/docs/idiomatic/index.html

Reply:
TBH, I don't remember, but it should definitely be updated.